README.md

```diff
@@ -133,12 +133,35 @@ pctl install --subscription-name pctl-profile \
              --profile-branch devel \
              --profile-url https://github.com/<usr>/<repo> \
              --profile-path bitnami-nginx \
-             --out install
+             --out <location of my flux repository>
 ```
 
 It's the user's responsibility to make sure that the local `git` setup has access to the url provided with `profile-url`.
 It can be any form of url as long as `git clone` understands it.
 
+#### Cloning private repository resources
+
+pctl clones all repository local resources and puts them into the flux repository. This is done so that the user doesn't
+have to include access credentials for all private repositories including nested profiles. However, for repository local
+resources to work, the user has to provide the location of the GitRepository object that flux manages.
```

Comment:
I feel like this line doesn't make it super clear that "provide us the gitrepository resource that points to the place where your installing your profile", but I have no idea how to word this nicely

Reply:
I'll figure something out.